ports/geckolib/properties.mako.rs

```diff
@@ -184,6 +185,54 @@ def set_gecko_property(ffi_name, expr):
     }
 </%def>
 
+<%def name="clear_color_flags(color_flags_ffi_name)">
+    % if color_flags_ffi_name:
+    self.gecko.${color_flags_ffi_name} &= !(gecko_style_structs::BORDER_COLOR_SPECIAL as u8);
+    % endif
+</%def>
+
+<%def name="set_current_color_flag(color_flags_ffi_name)">
+    % if color_flags_ffi_name:
+    self.gecko.${color_flags_ffi_name} |= gecko_style_structs::BORDER_COLOR_FOREGROUND as u8;
+    % else:
+    // FIXME(heycam): To handle this, we would need to get the
```

Comment:
Unless this is actionable (i.e. unless we know of properties that will hit this), I think we should remove the FIXME(heycam) prefix. Ideally I'd like to remove all FIXMEs from geckolib before we ship.